kymatio/scattering2d/frontend/numpy_frontend.py

```diff
@@ -14,8 +14,7 @@ def __init__(self, J, shape, L=8, max_order=2, pre_pad=False,
         ScatteringBase2D.create_filters(self)
 
     def scattering(self, input):
-        if not type(input) is np.ndarray:
```

Comment:
again the question whether this numpy file modification is an integral part of this PR